roles/analysis/README.md

```diff
@@ -1,25 +1,61 @@
+Metadata is required for leap preupgrade analysis to run.  When a system uses CDN, this is provided automatically.  For systems not on CDN, package metadata files needs to be provided by URL in leapp_metadata_url. Once obtained during IPU, file is unpacked on IPU host under `/etc/leapp/files` directory. 
+
+For more information, refer to the knowledge article [Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades (including Satellite)](https://access.redhat.com/articles/3664871).
 
 ## Role variables
 
-| Name                    | Default value         | Description                                         |
-|-------------------------|-----------------------|-----------------------------------------------------|
-| leapp_upgrade_type      | "satellite"           | Set to "cdn" for hosts registered with Red Hat CDN and "rhui" for hosts using rhui repos. |
-| leapp_metadata_url      |                       | See Requirements section above.                     |
-| leapp_answerfile        |                       | Optional multi-line string. If defined, this will be used as the contents of `/var/log/leapp/answerfile`. |
-| leapp_preupg_opts       |                       | Optional string to define command line options to be passed to the `leapp` command when running the pre-upgrade. |
+| Name                  | Type | Default value         | Description                                         |
+|-----------------------|------|-------------------------|-------------------------------------------------|
+| analysis_packages_el7 | List | leapp-upgrade             | RPMS that need to be installed for IPU to RHEL7 | 
+| analysis_repos_el7    | List | rhel-7-server-extras-rpms | Repo to be enabled for IPU to RHEL7             |
+| analysis_packages_el8 | List | leapp-upgrade             | RPMS that need to be installed for IPU to RHEL8 | 
+| analysis_repos_el8    | List | rhel-7-server-extras-rpms | Repo to be enabled for IPU to RHEL7             |
+| leapp_upgrade_type    | String  | disconnnected             | Whether connected to RHSM/CDN or not            |
+| leapp_preupg_opt      | String  | --no-rhsm when leapp_upgrade_type is connected | Upstream repository usage - whether to use RHSM | 
+| leapp_metadata_url      | String | | URL to the leapp metadata, usually over https   |
+| leapp_enable_repos_args | String | | --enablerepo (leapp_repos_enabled) or blank                         |
+| leapp_repos_enabled    | String | satellite-client-6-for-rhel-{{ ansible_distribution_major_version | int + 1 }}-x86_64-rpms | Satellite repo for the satellite client | 
```

Comment:
This is not rendering correctly because of the extra  `|` char.

Reply:
this looks to be rendering correctly from what I can tell

Reply:
Actually, looking deeper, I noticed the leapp_repos_enabled variable of String type with this default value is commented out in the code. As currently implemented, this variable is List type and default value is `[]`.

Reply:
That is correct, it is a list, and the default is an empty list.